chat-plugins/trivia.js

```diff
@@ -836,16 +843,25 @@ const commands = {
 			questions = sliceCategory(category);
 		}
 
-		if (questions.length < SCORE_CAPS[length] / 5) {
-			if (isRandom) return this.errorReply("There are not enough questions in the randomly chosen category to finish a trivia game.");
-			if (isAll) return this.errorReply("There are not enough questions in the trivia database to finish a trivia game.");
-			return this.errorReply("There are not enough questions under the category \"" + CATEGORIES[category] + "\" to finish a trivia game.");
-		}
-
 		// This prevents trivia games from modifying the trivia database.
 		questions = Object.assign([], questions);
 		// Randomizes the order of the questions.
 		questions = Dex.shuffle(questions);
```

Comment:
`Object.assign([], questions)` was dumb of me to write and should be `questions.slice()`

Shuffle the questions at the end of the function so you're not shuffling both Trivia and WL's questions when it's not needed though.